setup.py

```diff
@@ -31,6 +31,9 @@
     keywords='pydocstyle, PEP 257, pep257, PEP 8, pep8, docstrings',
     packages=('pydocstyle',),
     package_dir={'': 'src'},
+    install_requires=[
+        'jedi==0.9.0',
```

Comment:
Eventually this will not be pinned. I temporarily pinned this because a new `jedi` was released while I was working on this, which broke some of my changes.